### ISSUE TITLE
In step2_train_student.py

### ISSUE THREAD

**Issue by NeonLeexiang** [cvlab-yonsei/PISR#18]
In step2_train_student.py, you only set the teacher_model into eval type, I wonder whether the teacher model is trained while loss.backward(). Since the eval() only freeze the BN layer and Dropout layer, but the require_grad is still True.
